wellcomeml/ml/spacy_classifier.py

```diff
@@ -132,6 +132,42 @@ def fit(self, X, Y):
                 )
         return self
 
+    def partial_fit(self, X, Y, classes=None):
+        """
+        Args:
+            X: 1d list of documents
+            Y: 2d numpy array (nb_examples, nb_labels)
+        TODO: Generalise to y being 1d
+        """
+        Y = np.array(Y)
+
+        if not hasattr(self, 'unique_labels'):
+            # We could also use classes here
```

Comment:
Yes. Either add a class check, or remove the parameter from above? It seems it is not being used...

Reply:
The parameter is needed to support the sklearn API